source/common/grpc/json_transcoder_filter.cc

```diff
@@ -248,10 +250,11 @@ Http::FilterDataStatus JsonTranscoderFilter::decodeData(Buffer::Instance& data,
   const auto& request_status = transcoder_->RequestStatus();
 
   if (!request_status.ok()) {
-    ENVOY_LOG(debug, "Transcoding request error " + request_status.ToString());
+    ENVOY_LOG(debug,
+              "Transcoding request error " + ProtobufTypes::FromString(request_status.ToString()));
```

Comment:
Do you mean just can just have request_status without to and from string conversions?